installation/tmpl/remove/default.php

```diff
@@ -174,12 +178,14 @@ class="j-install-step-form flex-column mt-5 border rounded d-none"
 				</table>
 
 				<?php endif; ?>
-				<?php if ($this->development) : ?>
-					<div class="alert flex-column mb-1" id="removeInstallationTab">
-						<span class="mb-1 font-weight-bold"><?php echo Text::_('INSTL_SITE_DEVMODE_LABEL'); ?></span>
-						<button class="btn btn-danger mb-1" id="removeInstallationFolder"><?php echo Text::sprintf('INSTL_COMPLETE_REMOVE_FOLDER', 'installation'); ?></button>
-					</div>
-				<?php endif; ?>
+				<div class="alert flex-column mb-1" id="removeInstallationTab">
+					<?php if ($this->development) : ?>
+						<span class="mb-1 font-weight-bold"><?php echo Text::sprintf('INSTL_SITE_DEVMODE_LABEL', 'installation'); ?></span>
```

Comment:
```suggestion
						<span class="mb-1"><strong><?php echo Text::sprintf('INSTL_SITE_DEVMODE_LABEL', 'installation'); ?></strong></span>
```

Why would you want to change a language string to add html markup